src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1074,4 +1074,8 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Text_PackageLevelTopLevel" xml:space="preserve">
     <value>Top-level</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="Label_MultiplePackageVulnerableVersionToolTip" xml:space="preserve">
+    <value>This package installed versions: {0} contain vulnerabilities.</value>
```

Comment:
Fixing grammar. Suggesting alternative, but please check with @albarry4 on the wording.

```suggestion
    <value>These installed package versions contain vulnerabilities: {0}</value>
```